indico/MaKaC/webinterface/rh/base.py

```diff
@@ -843,7 +843,12 @@ def _checkProtection(self):
 class RHDisplayBaseProtected(RHProtected):
 
     def _checkProtection(self):
-        if not self._target.canAccess( self.getAW() ):
+        if isinstance(self._target, Conference):
+            can_access = self._target.canAccess(self.getAW())
+        else:
+            can_access = self._target.can_access(session.user)
```

Comment:
Does this work in all cases? `Abstract` has a `canAccess` too for example. I'd rather check whether the object has a `can_access` method instead (same below for `canModify`/`can_manage`)